### PR TITLE
atcore init command

### PR DESCRIPTION
**Pull request by msouff**

### CODE REVIEW

**Review by swainn — CHANGES_REQUESTED** [review]

File: tethysext/atcore/resources/xml_templates/layer_template.xml [hunk 1]
```diff
@@ -1,16 +1,32 @@
 <layer>
   <defaultStyle>
+  {%- if ':' in default_style %}
+    {% set style_workspace = default_style.split(':')[0] %}
+    {% set style_name = default_style.split(':')[1] %}
+    <name>{{ default_style }}</name>
```
[the remark this answers is not in the document]
Comment (msouff) [reply tethysext/atcore/resources/xml_templates/layer_template.xml:6]: No, default style is equal to workspace:style_name. Basically the fullname including the workspace.

File: tethysext/atcore/services/map_manager.py [hunk 2]
```diff
@@ -498,19 +498,23 @@ def get_map_extent(self):
 
         return view, extent
 
-    def generate_custom_color_ramp_divisions(self, min_value, max_value, num_divisions, first_division=1,
-                                             top_offset=0, bottom_offset=0, prefix='val'):
+    def generate_custom_color_ramp_divisions(self, min_value, max_value, num_divisions, value_precision=2,
```
[the remark this answers is not in the document]
Comment (msouff) [reply tethysext/atcore/services/map_manager.py:501]: But no app is using the function like that yet. Agwa is the only other app using this (at least that we know of) and it does use kwargs. Our criteria was that these values are related to each other (min_value, max_value, num_divisions, value_precision=2), having value_precision at the end makes it seem like it is unrelated.

That's the why. Let us know if you still want us to move value_precision to the end.

File: tethysext/atcore/services/map_manager.py [hunk 3]
```diff
@@ -528,10 +532,10 @@ def generate_custom_color_ramp_divisions(self, min_value, max_value, num_divisio
         b = max_val - (m * max_div)
 
         for i in range(min_div, max_div + 1):
-            if (max_value - min_value) / num_divisions <= 2:
-                divisions['{}{}'.format(prefix, i)] = "{0:.5f}".format(m * i + b)
-            else:
-                divisions['{}{}'.format(prefix, i)] = ceil(m * i + b)
+            divisions[f'{prefix}{i}'] = f"{(m * i + b):.{value_precision}f}"
```
Comment (msouff) [inline tethysext/atcore/services/map_manager.py:535]: BTW, look at this beauty. You can have nested variables within an f string.

Comment (swainn) [reply tethysext/atcore/services/map_manager.py:535]: Nice

File: tethysext/atcore/services/map_manager.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (swainn) [reply tethysext/atcore/services/map_manager.py:501]: If you are sure.

**Review by swainn — APPROVED** [review]

**Pull request merged**